### PR TITLE
Asintel 1016 resilience layer views

### PR DESCRIPTION
**Pull request by sarmaevanidfe**
Resilience layer views 
[ASData_PL].[DAS_SpendControl_V2]
[ASData_PL].[DAS_SpendControlNonLevy_V2]
[ASData_PL].[DAS_TransactionLine_V2]
[Data_Pub].[DAS_Commitments_V2]
[Data_Pub].[DAS_Employer_Account_Transfers_V2]
[Data_Pub].[DAS_Employer_Accounts_V2]
[Data_Pub].[DAS_Employer_AccountTransactions_V2]
[Data_Pub].[DAS_Employer_Agreements_V2]
[Data_Pub].[DAS_Employer_LegalEntities_V2]
[Data_Pub].[DAS_Employer_PayeSchemes_V2]
[Data_Pub].[DAS_Employer_Transfer_Relationship_V2]
[Data_Pub].[DAS_LevyDeclarations_v2]
[Data_Pub].[DAS_Payments_V2]

### CODE REVIEW

File: src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql [hunk 1]
```diff
@@ -106,6 +106,7 @@ VALUES
...
 ,('Commitments','Apprenticeship','dbo','[Id],[CommitmentId],[TrainingType],[TrainingCode],[TrainingName],[Cost],[StartDate],[EndDate],[AgreementStatus],[PaymentStatus],[EmployerRef],[ProviderRef],[CreatedOn],[AgreedOn],[PaymentOrder],[StopDate],[PauseDate],[HasHadDataLockSuccess],[PendingUpdateOriginator],[EPAOrgId],[CloneOf],[ReservationId],[IsApproved],[CompletionDate],[ContinuationOfId],[MadeRedundant],[OriginalStartDate],[Age]','[NINumber]','[FirstName],[LastName],[DateOfBirth],[ULN]','comt_Apprenticeship',0)
+,('Commitments','Providers','dbo','[Ukprn], [Name], [Created], [Updated]','','','Comt_Providers',0)
```
Comment (arvinthseran) [inline src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql:109]: I believe the columns [Ukprn], [Name], should be under Columns to Mask?

**Review by arvinthseran — CHANGES_REQUESTED** [review]

**Review by arvinthseran — APPROVED** [review]

**Review by uhimabindu — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.DATA.MANAGEMENT/Views/ASDataPL_DAS_SpendControl_V2.sql [hunk 2]
```diff
@@ -0,0 +1,149 @@
+﻿    CREATE VIEW [ASData_PL].[DAS_SpendControl_V2]
+	AS
+			SELECT 
+				 COALESCE(Account.EmployerAccountId, -1)                                   AS EmployerAccountId
+				,COALESCE(Account.DasAccountId, 'XXXXXX')                                  AS DasAccountId
+				,COALESCE(Account.DasAccountName, 'NA')                                    AS DasAccountName
```
[the remark this answers is not in the document]
Comment (sarmaevanidfe) [reply src/SFA.DAS.DATA.MANAGEMENT/Views/ASDataPL_DAS_SpendControl_V2.sql:6]: in the sub query both DASAccountName and LegalEntityName are updated as 'Redacted' for those two views

[the remark this answers is not in the document]
Comment (uhimabindu) [reply src/SFA.DAS.DATA.MANAGEMENT/Views/ASDataPL_DAS_SpendControl_V2.sql:6]: Thanks, That's all looking then.

**Review by uhimabindu — APPROVED** [review]
Approved

**Pull request merged**